sw/tests/helloworld.c

```diff
@@ -16,7 +16,7 @@
 int main(void) {
     char str[] = "Hello World!\r\n";
     uint32_t rtc_freq = *reg32(&__base_regs, CHESHIRE_RTC_FREQ_REG_OFFSET);
-    uint64_t reset_freq = clint_get_core_freq(rtc_freq, 2500);
+    uint64_t reset_freq = clint_get_core_freq(rtc_freq, 2048);
```

Comment:
Perche?

Reply:
I ran the helloworld test to ensure my changes didn't break Cheshire. With a setting of 2048, the UART has the correct baud rate, and I was able to see the message on the output terminal. Otherwise, it does not work. It depends on the clock frequency set in the TB. I had already experienced this issue while working on Chimera, and we solved it in exactly the same way.